configure.ac

```diff
@@ -218,6 +218,7 @@ AS_IF([test "x$with_docs_only" = xyes],
      AM_CONDITIONAL([HAVE_AARCH64_HI1620], [false])
      AM_CONDITIONAL([HAVE_FUSE3], [false])
      AM_CONDITIONAL([IODEMO_CUDA], [false])
+     AM_CONDITIONAL([HAVE_SIGACTION_OVERRIDE], [false])
```

Comment:
let's remove this conditial
we added it initially to disable signal override in libucs.so, now it's a separate lib -- no longer needed

Reply:
> let's remove this conditial
> we added it initially to disable signal override in libucs.so, now it's a separate lib -- no longer needed

removed configuration